### DISCUSSION TITLE
Lesson 4: Solcx Compile Issue

### DISCUSSION THREAD

**Discussion by vicshi06** [smartcontractkit/full-blockchain-solidity-course-py#793] [Q&A] [answered]
Hi,

I have successfully downloaded the Solcx as you can see in the terminal, but I don't know why it doesn't read my solcx and cant run my code. 

![image](https://user-images.githubusercontent.com/90171690/149614892-cefd457f-a38c-45e2-b4a7-362b37bc926b.png)

**Comment by bzpassersby** [comment, marked as answer]
I think you need to add a line: install_solc("0.6.0") after your first line, it's also suggested in the terminal. 
See updated code here:
https://github.com/smartcontractkit/full-blockchain-solidity-course-py/blob/main/chronological-issues-from-video.md

**Reply by vicshi06** [reply]
Thanks

**Reply by cromewar** [reply]
You are awesome @bzpassersby !!
